.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -43,4 +43,4 @@ password for all users: 12345678
 
 ### Questions? Join Slack!
 
-We highly recommend that you join us in slack https://rubyforgood.herokuapp.com/ #casa channel to ask questions quickly and hear about office hours (currently Tuesday 5-7pm Pacific), stakeholder news, and upcoming new issues.
+We highly recommend that you join us in slack (http://bit.ly/3Quxc1Q) #casa channel to ask questions quickly and hear about office hours (currently Tuesday 5-7pm Pacific), stakeholder news, and upcoming new issues.
```

Comment:
I'm not exactly sure whether this link was attempting to go to slack directly in some way or be another link to the slack invitation. Either way it was a broken link so I changed it to be the slack invite link since it's what I had. Feel free to suggest something else.

<img width="997" alt="image" src="https://github.com/rubyforgood/casa/assets/444766/dacb4ae8-dacd-46e9-b47b-52cb419c7bd0">